src/pages/Rockets.js

```diff
@@ -0,0 +1,29 @@
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getrocket } from '../redux/rocketsSlice';
+import Rocket from '../components/SingleRocket';
+
+const Rockets = () => {
+  const RocketList = useSelector((state) => state.rocket);
```

Comment:
- [ ] Your rockets page is currently displaying a blank page signifying that the data is not being fetched from the API. Kindly check it out and ensure you are passing the right data to the API

![image](https://user-images.githubusercontent.com/63473584/225763445-0ae4486a-8a38-4720-af51-21c9f79585e8.png)

![image](https://user-images.githubusercontent.com/63473584/225763988-0febf0d3-ef21-43a4-86b2-1e015f00737a.png)

![image](https://user-images.githubusercontent.com/63473584/225764294-3afee736-5fb8-4d1e-a7c8-f1bc441ae1d7.png)

Reply:
thank you @vickymarz , we fixed the problem.

Reply:
-  Great job team. This has been well resolved.
<img width="1192" alt="Screenshot 2023-03-16 at 23 52 21" src="https://user-images.githubusercontent.com/58692590/225770348-55af5bb8-c764-4d37-b0c2-8cdcd4eb8f77.png">

<img width="1302" alt="Screenshot 2023-03-16 at 23 53 57" src="https://user-images.githubusercontent.com/58692590/225770512-cf296d9a-b7fc-4529-80ac-7c40ba8da60d.png">

<img width="1224" alt="Screenshot 2023-03-16 at 23 54 19" src="https://user-images.githubusercontent.com/58692590/225770541-872f2bc3-daef-402c-a38d-a6ba10bae175.png">